src/structures/Guild.js

```diff
@@ -634,9 +634,11 @@ class Guild {
   }
 
   /**
+   * Sets the position of the guild in the guild listing.
+   * <warn>This is only available when using a user account.</warn>
    * @param {number} position Absolute or relative position
    * @param {boolean} [relative=false] Whether to position relatively or absolutely
-   * @returns {Promise<Guild>}
+   * @returns {Promise<Guild>} This guild
```

Comment:
We never tag return values except they need explicit explanation, which in this case it really doesn't.